compiler/noirc_frontend/src/parser/parser/item.rs

```diff
@@ -134,52 +140,52 @@ impl<'a> Parser<'a> {
             self.comptime_mutable_and_unconstrained_not_applicable(modifiers);
 
             let use_tree = self.parse_use_tree();
-            return vec![ItemKind::Import(use_tree, modifiers.visibility)];
+            return vec![(ItemKind::Import(use_tree, modifiers.visibility), cfg_feature_disabled)];
```

Comment:
I wonder if it'd be more clear to have these as their own item kinds that wrap existing item kinds? That way we can limit touching external logic. We would likely need a little pass to go through and handle these though before e.g. imports or global items are processed. Perhaps it'd be cleaner to move the filtering of these to a small preprocessing pass.